test/e2e/metrics_test.go

```diff
@@ -36,16 +37,16 @@ var (
 var _ = Describe("Metrics", Label("metrics"), func() {
 	Context("When a metricpipeline exists", Ordered, func() {
 		var (
-			pipelines          *kyma.PipelineList
-			urls               *urlprovider.URLProvider
-			mockDeploymentName = "metric-receiver"
-			mockNs             = "metric-mocks"
+			pipelines       *kyma.PipelineList
+			urls            *urlprovider.URLProvider
+			mockBackendName = "metric-receiver"
+			mockNs          = "metric-mocks"
 		)
 
 		BeforeAll(func() {
 			k8sObjects, urlProvider, pipelinesProvider := makeMetricsTestK8sObjects(
```

Comment:
after splitting the metrics and traces e2e tests in multiple files, the backend creation will be done in the makeTestObjects functions and not anymore passed as an argument (like here). This is already done for all logging tests